service/controller/key/key_test.go

```diff
@@ -269,7 +269,7 @@ func Test_Functions_for_DNSKeys(t *testing.T) {
 }
 
 func Test_MasterNICName(t *testing.T) {
-	expectedMasterNICName := "3p5j2-Master-1-NIC"
+	expectedMasterNICName := "3p5j2-master-3p5j2-nic"
```

Comment:
This is different compared to the source code

https://github.com/giantswarm/azure-operator/blob/4337e788945831cbcb691baf47abe64c050fd164/service/controller/key/key.go#L390

```suggestion
	expectedMasterNICName := "3p5j2-Master-3p5j2-nic"
```

Reply:
good catch